tasks.py

```diff
@@ -181,6 +181,6 @@ def generate_infrahubctl(context: Context) -> None:
 
 
 @task(name="generate-sdk")
-def generate_python_sdk(context: Context) -> None:
+def generate_python_sdk(context: Context) -> None:  # noqa: ARG001
```

Comment:
you can remove context if we don't use it

Reply:
It seems to be required by `invoke` tasks